sklearn/neighbors/lof.py

```diff
@@ -192,8 +194,14 @@ def fit(self, X, y=None):
 
         self.negative_outlier_factor_ = -np.mean(lrd_ratios_array, axis=1)
 
-        self.threshold_ = -scoreatpercentile(
-            -self.negative_outlier_factor_, 100. * (1. - self.contamination))
+        if self.contamination is None:
+            self.threshold_ = -1.1  # inliers score around 1.
```

Comment:
1.5 is used in the paper in the experiments section. But it does not work on small dataset (the added tests break). I can put 1.5 and remove the added test in the case `contamination is None`.